jolt-core/src/test/resources/json/modifier/functions/mathTests.json

```diff
@@ -8,6 +8,14 @@
         },
         "data3": {
             "value": [-1,2,3.0]
+        },
+        "data4": {
+            "value": [5, 2],
```

Comment:
comment what each of these is testing.

Also need a test where "=div" is running off two params instead of an array.